dronegen: Switch linux-based push builds to GitHub

camscale opened the pull request:
Change the drone pipelines for linux-based push builds to call a GitHub 
actions workflow instead of running on drone runners. This includes one of
the builds for Windows which is done in a Linux container.

camscale committed:
dronegen: Switch linux-based push builds to GitHub

Change the drone pipelines for linux-based push builds to call a GitHub
actions workflow instead of running on drone runners. This includes one
of the builds for Windows which is done in a Linux container.

The old push pipelines that run the build on drone runners is now
removed as it is no longer used.

camscale committed:
ci: Update .drone.yml

This updates the linux-based push pipelines to delegate to github
actions.

camscale commented:
In draft until I fix the lint errors. I just need to remove the now-unused code for the old pipelines.

camscale committed:
Update e ref for push linux workflows

r0mant reviewed (commented):
Same question as in the other PR: can you run a full build/promotion cycle on a dev tag and link it here?

camscale commented:
> Same question as in the other PR: can you run a full build/promotion cycle on a dev tag and link it here?

https://drone.platform.teleport.sh/gravitational/teleport/25960 is the test build I did. Not a tag build since these are push pipelines. The drone run was just for the pipelines I changed an no others, since I had to dummy up the branch to have the push build fire and I only did that for the pipelines I changed.
